pkg/constants/featureflags.go

```diff
@@ -7,6 +7,10 @@ package constants
 const (
 	// FeatureContextCommand determines whether to surface the context command. This is disabled by default.
 	FeatureContextCommand = "features.global.context-target-v2"
+
+	// FeatureContextScopedPluginDiscoveryForTanzuContext determines whether to enable context-scoped plugin discovery for Tanzu context.
+	// This is disabled by default
+	FeatureContextScopedPluginDiscoveryForTanzuContext = "features.global.context-plugin-discovery-for-tanzu-context"
```

Comment:
Please review the naming of the feature-flag.

Reply:
how about just `features.global.plugin-discovery-for-tanzu-context`?